### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by zebra0303** [yujinakayama/atom-lint#143]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.

```
Atom.Object.defineProperty.get (/opt/atom/resources/app/src/atom.js:54:11)
Object.enable (/home/larry/.atom/packages/atom-lint/lib/atom-lint.coffee:28:35)
```

**Comment by jackmaney** [comment]
I don't know about anyone else, but these deprecation issues have caused atom-lint to completely stop linting on my system (in particular, no errors are shown, even if I purposefully introduce some).

**Comment by fxgear-dsc** [comment]
Hi Jackmaney,

Thanks for your reply.

I'm using Atom on Ubuntu 14.04 and following is my atom screen shot.
http://i.imgur.com/bEkBDni.png

-Larry

**Comment by zebra0303** [comment]
Hi Jackmaney,

Thanks for your reply.

I'm using Atom on Ubuntu 14.04 and following is my atom screen shot.
http://i.imgur.com/bEkBDni.png

-Larry

**Comment by jackmaney** [comment]
Larry,

Sorry, I'm not familiar with this codebase, and am neither a maintainer nor a developer for it. I'm just adding to your issue and stating that atom-lint doesn't work at all for me (no linting whatsoever).

**Comment by zebra0303** [comment]
Oh got it.

**Comment by jackmaney** [comment]
Honestly, I've moved on to [linter](https://atom.io/packages/linter), since atom-lint has completely stopped linting (even reinstalling Atom and installing atom-lint from scratch didn't help).
